README.md

```diff
@@ -1 +1,79 @@
-# configuration-caas
+# Cluster-as-a-Service base configuration
+
+This repo provides a configuration for a Cluster-as-a-Service, built on top of
+Crossplane. This configuration exposes a simple API to your internal developers
+for creating Kubernetes clusters. In the background, this configuration can create clusters in AWS, Azure, and GCP. It automatically applies common recommended practices, such as connecting the cluster to existing Flux deployments, automating VPC setup, and creating a Flux operator in a cluster.
+
+This repo is a starting point for you to deliver your own
+Cluster-as-a-Service. Fork this repository and customize the configuration to
+meet your teams' needs.
+
+To deploy a multicloud CaaS control plane in your organization, check out the
+[Multicloud Quickstart](https://docs.upbound.io/quickstart/multicloud-deploy/).
+
+Advantages of CaaS:
+
+- GitOps workflow
+- Production-ready template
+- Scalable architecture
+- Product agnostic approach
+
+## Available APIs
+
+This repository uses Compositions for AWS, Azure, and GCP provider APIs, as well as the Upbound Control Plane provider. For more information, review the provider Configurations below:
+
+- [`Cluster.aws.caas.upbound.io`](https://marketplace.upbound.io/configurations/upbound/configuration-caas/v0.1.0/resources/aws.caas.upbound.io/XCluster/v1alpha1) 
+    - Provision/Manage an EKS Cluster
+
+- [`Cluster.azure.caas.upbound.io`](https://marketplace.upbound.io/configurations/upbound/configuration-caas/v0.1.0/resources/azure.caas.upbound.io/XCluster/v1alpha1) 
+
+    - Provision/Manage an AKS Cluster
+    
+- [`Cluster.gcp.caas.upbound.io`](https://marketplace.upbound.io/configurations/upbound/configuration-caas/v0.1.0/resources/gcp.caas.upbound.io/XCluster/v1alpha1) 
+    - Provision/Manage a GKE Cluster
+
+- [`ControlPlane.mcp.caas.upbound.io`](https://marketplace.upbound.io/configurations/upbound/configuration-caas/v0.1.0/resources/mcp.caas.upbound.io/XControlPlane/v1alpha1)  
+    - Provision/Manage an Upbound Control Plane
+
+- [`Connector.mcp.caas.upbound.io`](https://marketplace.upbound.io/configurations/upbound/configuration-caas/v0.1.0/resources/mcp.caas.upbound.io/XCluster/v1alpha1)
+    - Provision/Manage an MCP Connector
+
+Each API in the [`apis`](https://github.com/upbound/configuration-caas/tree/main/apis) folder contains compositions and definitions for each cloud provider. Once you clone the CaaS repository, you can modify these compositions to fit your organizations needs. 
```

Comment:
> folder contains compositions and definitions

Potential nit: each API definitionally _is_ a CompositeResourceDefinition (schema) and one more compositions (implementation). Do you think we could further clarify this for new users?

Reply:
Not a nit! I need to memorize the terminology 😃 . Attempted an update to clarify but feel free to update.